### ISSUE TITLE
Error in SQLPAGE_SITE_PREFIX if used - ( hyphen )

### ISSUE THREAD

**Issue by rinshadka** [sqlpage/SQLPage#689]
when the site prefix is having ` -` and ` _` , then it will break if we refresh the same via browser.

 Instead if I use `%2D` in the place of `-` ( hyphen) in browser address bar , it will work again

Any workaround for this one. I am not using nginx proxy. direct sqlpage with site prefix is having this issue

To reproduce:

```
git clone https://github.com/sqlpage/SQLPage

cd SQLPage/examples/official-site/

SQLPAGE_SITE_PREFIX="/myapp-myapp-myapp" SQLPAGE_PORT=5006 SQLPAGE_HOST=10.10.11.248 sqlpage
```

![image](https://github.com/user-attachments/assets/d78e2a76-9b59-4204-9e03-8d32317e7ecd)

But If I hit an enter by just clicking the URL in address bar

![image](https://github.com/user-attachments/assets/a0ee8c4f-5619-48cf-8b98-aff71cbb6e10)

Thanks.

**Comment by lovasoa** [comment]
Thank you for the report ! I'll look into it !

**Comment by lovasoa** [comment]
Hello ! I justr tried and cannot reproduce the error. Can you copy and paste the full error message here ?

**Comment by rinshadka** [comment]
An error occurred while trying to handle your request: Unable to get SQL file "/myapp-myapp-myapp/index.sql": Couldn't load "/myapp-myapp-myapp/index.sql" into cache: Unsupported path: "/myapp-myapp-myapp/index.sql". Path component 'RootDir' is not allowed.

**Comment by rinshadka** [comment]
If I use the url: `http://10.10.11.248:5006` , then I get 309 redirected to` http://10.10.11.248:5006/myapp-myapp-myapp/`

But if I directly hit the `http://10.10.11.248:5006/myapp-myapp-myapp/` , then I will land in the above error.

**Comment by lovasoa** [comment]
Thank you ! I can reproduce it now. I'll make a fix for v0.31. 

Thanks again for helping make sqlpage better !

**Comment by rinshadka** [comment]
> Thank you ! I can reproduce it now. I'll make a fix for v0.31.
> 
> Thanks again for helping to make sqlpage better :)

Thanks you.. :) when can I expect this release to publish..
